ui/app/src/components/PreviewConcierge/PreviewConcierge.tsx

```diff
@@ -563,10 +563,12 @@ export function PreviewConcierge(props: PropsWithChildren<{}>) {
       switch (type) {
         case guestSiteLoad.type:
         case guestCheckIn.type:
-          const { version: guestVersion } = payload;
+          const { version: guestFullVersion } = payload;
           const studioVersion = env.version.slice(0, 5);
-
-          if (type === guestCheckIn.type && guestVersion?.slice(0, 5) !== studioVersion) {
+          const guestVersion = guestFullVersion.slice(0, 5);
+          const isCompatibleVersion =
+            guestVersion === studioVersion || (studioVersion === '4.1.0' && guestVersion === '4.0.3');
+          if (type === guestCheckIn.type && !isCompatibleVersion) {
```

Comment:
How about we simply do: 
```suggestion
          if (type === guestCheckIn.type && ['4.1.0', '4.0.3'].includes(payload.version.slice(0, 5));
```
Don't even need studio version as things stand do we?

Reply:
We don't need it for now